diff --git a/drivers/usb/core/quirks.c b/drivers/usb/core/quirks.c
index 17c3d472fb1304ce10164124129d94bda512c379..f99a65a64588fe9b4deca18c593bfe0f690726f5 100644
--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -519,9 +519,6 @@ static const struct usb_device_id usb_quirk_list[] = {
 	/* INTEL VALUE SSD */
 	{ USB_DEVICE(0x8086, 0xf1a5), .driver_info = USB_QUIRK_RESET_RESUME },
 
-	/* Quectel EG25G Modem */
-	{ USB_DEVICE(0x2c7c, 0x0125), .driver_info = USB_QUIRK_RESET },
-
 	{ }  /* terminating entry must be last */
 };
 
